operator-common/src/test/java/io/strimzi/operator/common/operator/resource/AbstractCustomResourceOperatorIT.java

```diff
@@ -114,53 +113,45 @@ public void after() {
     @Test
     public void testUpdateStatus(VertxTestContext context) {
         String resourceName = getResourceName(RESOURCE_NAME);
-        Checkpoint async = context.checkpoint();
         String namespace = getNamespace();
 
         CrdOperator<C, T, L> op = operator();
 
         LOGGER.info("Getting Kubernetes version");
         PlatformFeaturesAvailability.create(vertx, client)
-                .onComplete(context.succeeding(pfa -> context.verify(() -> {
-                    assertThat("Kubernetes version : " + pfa.getKubernetesVersion() + " is too old",
-                            pfa.getKubernetesVersion().compareTo(KubernetesVersion.V1_16), CoreMatchers.is(not(lessThan(0))));
-                })))
+                .onComplete(context.succeeding(pfa -> context.verify(() ->
+                        assertThat("Kubernetes version : " + pfa.getKubernetesVersion() + " is too old",
+                        pfa.getKubernetesVersion().compareTo(KubernetesVersion.V1_16), CoreMatchers.is(not(lessThan(0)))))))
```

Comment:
Can we indent this line to make it more readable?

Reply:
:+1: